### DISCUSSION TITLE
Chmod error don't understand

### DISCUSSION THREAD

**Discussion by denizturgut** [community/community#27180] [Actions]
<p>I’m following this <a href="https://engineering.talkdesk.com/test-and-deploy-an-ios-app-with-github-actions-44de9a7dcef6" rel="noopener nofollow ugc">tutorial</a> for CD but got chmod error return. I tried to look for the usage and error but still don’t understand what it means. Was there a syntax problem or incorrect path? Thank you.</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/4/3/434ec801a28959df16bd54c6c24fc0b37bedb6a2.png" data-download-href="/uploads/short-url/9BqMh1IhjXDPxiTjPlLbEvIJTZ8.png?dl=1" title="Screen Shot 2020-12-20 at 12.37.18 AM" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/52680799/181102170-0258ad17-4c5d-4e92-b42d-71f4ad543373.png" alt="Screen Shot 2020-12-20 at 12.37.18 AM" data-base62-sha1="9BqMh1IhjXDPxiTjPlLbEvIJTZ8" width="690" height="265" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/4/3/434ec801a28959df16bd54c6c24fc0b37bedb6a2_2_690x265.png, https://aws1.discourse-cdn.com/github/optimized/3X/4/3/434ec801a28959df16bd54c6c24fc0b37bedb6a2_2_1035x397.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/4/3/434ec801a28959df16bd54c6c24fc0b37bedb6a2_2_1380x530.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/4/3/434ec801a28959df16bd54c6c24fc0b37bedb6a2_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screen Shot 2020-12-20 at 12.37.18 AM</span><span class="informations">1504×579 49.7 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by airtower-luna** [comment, marked as answer]
<p>It’s a syntax error. Chmod is used to change filesystem permissions, and you need to give it the permission change you want and the file(s) to modify, in that order. The permission specification is missing in your command.</p>
<p>On the other hand, sudo shouldn’t be necessary to modify a file from your repository.</p>

**Comment by denizturgut** [comment]
<p>Thank you. I did chmod 755 to give the permission and it works.</p>
